commands/questions/create-questions.js

```diff
@@ -1,51 +1,53 @@
-// Import the required modules
+/* eslint-disable no-console */
 const chalk = require('chalk');
 const figlet = require('figlet');
 const { validFileName, validPath } = require('./validate.js');
 
 // Print a header for the application in the terminal
 console.log(
   chalk.yellow(
-    figlet.textSync('Ontario.ca Toolkit', { horizontalLayout: 'full' })
-  )
+    figlet.textSync('Ontario\nFrontend', { horizontalLayout: 'full' }),
+  ),
 );
 
 // Define the questions for the inquirer prompt
 const createQuestions = [
-  {
-    type: "input",
-    name: "projectName",
+  { // cursor problem
```

Comment:
The addition of a comment is good here, but it could use a bit of an explanation like "If a prompt exceeds 1 line and wraps it can cause the input cursor to be misaligned. Move input under question, instead of inline."